docs/examples/KcardGrid/BasicGrids/1-2-2.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+
+  <KCardGrid
+    layout="1-2-2"
+    :skeletonsConfig="skeletonsConfig2"
```

Comment:
- Since now each example lives in it's own vue component file, we do not use to use names like 'skeletonConfig2' like we were using earlier. 
- We can now simply replace it with a generic name like 'skeletonsConfig'